libs/ui/src/admin/ArtApprovalModal/ArtApprovalForm.tsx

```diff
@@ -0,0 +1,136 @@
+import { FC, memo, useState } from 'react'
+
+import '@splidejs/splide/dist/css/themes/splide-default.min.css'
+import {
+  Avatar,
+  HStack,
+  Text,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalOverlay,
+  Stack,
+  Box,
+  Image,
+  Flex,
+  SimpleGrid,
+  Button,
+  Textarea,
+} from '@chakra-ui/react'
+import { Splide, SplideSlide } from '@splidejs/react-splide'
+
+import { ArtFeedbackForm } from './ArtFeedbackForm'
+import { ArtImageProps, ArtApprovalFormTypes } from './types'
+
+const ArtImage: FC<ArtImageProps> = memo(({ image, alt }) => {
+  return (
+    <Image
+      objectFit="contain"
+      src={`${process.env['NX_API_URL']}${image.url}`}
+      alt={alt}
+    />
+  )
+})
+
+export const ArtApprovalForm: FC<ArtApprovalFormTypes> = ({
+  onReject,
+  onApprove,
+  onDelete,
+  artId,
+  artDescription,
+  artTitle,
+  artistAvatar,
+  artImages,
+  editorId,
+  isOpen,
+  onClose,
+  editorAvatar,
+  editorName,
+  artistName,
+  onSave,
+  setIsEditing,
+  isEditing,
+}) => {
+  const [description, setDescription] = useState(artDescription)
+  return (
+    <Box>
+      <Modal onClose={onClose} size="6xl" isOpen={isOpen}>
+        <ModalOverlay />
+        <ModalContent>
+          <ModalCloseButton />
+          <ModalBody>
+            <SimpleGrid columns={{ base: 1, lg: 2 }} spacing={4}>
+              <Stack>
+                {/*TODO Image should has zoom  */}
+                {artImages && artImages.length > 1 ? (
+                  <Splide>
+                    {artImages?.map((image, index) => (
+                      <SplideSlide key={index}>

```

Comment:
Can you confirm if it's better to have centered images?

```suggestion
                      <Center as={SplideSlide} key={index}
```

Reply:
When we give Center, Splide component doesn't working.

Reply:
I mean `<Center as={SplideSlide} key={index}>` not only `<Center key={index}>`

Reply:
Yes I tried something. But splide css didn't work.